packages/python-pywebcopy/PKGBUILD

```diff
@@ -5,14 +5,13 @@ pkgname=python-pywebcopy
 _pkgname=pywebcopy
 pkgver=7.0.2
 _pyver=3.12
-pkgrel=4
+pkgrel=5
 pkgdesc='Python package for cloning complete webpages and websites to local storage.'
 arch=('any')
 url='https://pypi.org/project/pywebcopy/#files'
 license=('Apache')
-depends=('python' 'python-requests' 'python-lxml' 'python-pyquery'
-         'python-parse' 'python-w3lib' 'python-beautifulsoup4' 'python-fire')
-makedepends=('python-setuptools' 'python-pip' 'python-wheel')
+depends=('python' 'python-cachecontrol' 'python-lxml-html-clean' 'python-requests' 'python-six')
```

Comment:
I didn't try without it but I added because it was in requirements.

Reply:
The developer said that six is needed to make pywebcopy compatible with Python2.7 (deprecated). We can keep it in case there is an old package based on Python2.7 that uses pywebcopy as dependency.